subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/SamplesMavenPublishIntegrationTest.groovy

```diff
@@ -109,38 +109,6 @@ class SamplesMavenPublishIntegrationTest extends AbstractSampleIntegrationTest {
         dsl << ['groovy', 'kotlin']
     }
 
-    @Unroll
-    @UsesSample("maven-publish/multiple-publications")
-    def "multiple publications with #dsl dsl"() {
```

Comment:
:o: We could keep the test but only check the one customised publication, right? I think this is actually valuable to test that the documentation is documenting what it says (publishing a jar with the expected name)

Reply:
Good idea. I refocused the test instead now on this changed use-case.